client/components/Message.vue

```diff
@@ -40,6 +40,13 @@
 					<span class="only-copy">&gt; </span>
 				</template>
 			</span>
+			<span v-else-if="message.type === 'plugin'" class="from">
+				<template v-if="message.from && message.from.nick">
+					<span class="only-copy">[</span>
+					*{{ message.from.nick }}
```

Comment:
We can style this nicer, give it an icon with FA for example.

Reply:
![https://i.imgur.com/TUduR8L.png](https://i.imgur.com/TUduR8L.png)